src/pages/case-study/doctorlink-setting-up-a-design-system.js

```diff
@@ -0,0 +1,674 @@
+import React from 'react'
+import { graphql } from 'gatsby'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+
+import { Grid, Row, Col } from '../../components/grid'
+import BlueBackground from '../../components/Common/BlueBackground'
+import GreyBackground from '../../components/Common/GreyBackground'
+import Layout from '../../components/layout'
+import Head from '../../components/Common/Head'
+import CaseStudyHero from '../../components/Common/CaseStudyCards/CaseStudyHero'
+import {
+  renderImage,
+  renderText,
+  TextColumnsBlock,
+  FullWidthBlock,
+  VideoBlock,
+  ImagesBlock,
+  TextAndImageBlock,
+  TextAndResizedImageBlock
+} from '../../components/Common/CaseStudyCards/CaseStudyBlocks'
+// import SubtitleWithBody from '../../components/Common/SubtitleWithBody'
+import Statement from '../../components/Common/Statement'
+import CaseStudyPreview from '../../components/Common/CaseStudyCards/CaseStudyPreview'
+
+const BlockRow = styled(Row)`
```

Comment:
@omonk Would you like me to move the `BlockRow` component to `./src/components/Common/CaseStudyCards/CaseStudyBlocks.js`. Maybe something re-usable for you with Central working?

Reply:
No worries for now, I will move a few bits of pieces around in my central working branch after this is merged and I have rebased